manifests/03-role.yaml

```diff
@@ -100,6 +100,20 @@ rules:
 - apiGroups:
   - rbac.authorization.k8s.io
   resources:
+  - clusterroles
   - clusterrolebindings
   verbs:
   - "*"
+---
+kind: ClusterRoleBinding
```

Comment:
for consistency, can you move this to `manifests/04-role-binding.yaml`?

Reply:
@ewolinetz updated